istio-control/istio-config/templates/deployment.yaml

```diff
@@ -181,9 +181,28 @@ spec:
             readOnly: true
           - name: envoy-config
             mountPath: /var/lib/istio/galley/envoy
+          {{- if .Values.global.sds.enabled }}
+          - name: sds-uds-path
+            mountPath: /var/run/sds
+            readOnly: true
+          - name: istio-token
+            mountPath: /var/run/secrets/tokens
```

Comment:
will this not collide with K8s's `/var/run/secrets/tokens` mount?

Reply:
Good question. This one is to load the trustworthy JWT to the "tokens" path. The default K8s JWT is in the "kubernetes.io/serviceaccount" path. I don't see they collide since they are using different sub-directories.